docs/zh-cn/2.0.1/user_doc/architecture/design.md

```diff
@@ -68,7 +68,7 @@
 
 * **UI** 
 
-    系统的前端页面，提供系统的各种可视化操作界面，详见<a href="/zh-cn/docs/2.0.1/user_doc/system-manual.html" target="_self">系统使用手册</a>部分。
+    系统的前端页面，提供系统的各种可视化操作界面，详见[功能介绍](../guide/homepage.md)部分。
```

Comment:
I found it also in dev doc https://github.com/apache/dolphinscheduler-website/blob/d423ef4bf7b9a7a6f48cf5bab5d6a100d42122a1/docs/zh-cn/dev/user_doc/architecture/design.md#L102. could your also change `dev` zh and en doc? @uestczhuhao

Reply:
done @zhongjiajie